src/docs/development/tools/android-studio.md

```diff
@@ -87,88 +87,78 @@ The Dart plugin performs code analysis that enables:
-    The bottom **Run** pane shows logs output:<br>
-
-    ![Log pane]({% asset tools/android-studio/log.png @path %})
+    The bottom **Run** pane shows logs output.
 
 ### Run app with breakpoints
 
  1. If desired, set breakpoints in your source code.
  1. Click the **Debug icon** in the toolbar, or invoke **Run > Debug**.
     * The bottom **Debugger** pane shows Stack Frames and Variables.
     * The bottom **Console** pane shows detailed logs output.
-    * Debugging is based on a default launch configuration. To customize this,
-      click the drop-down button to the right of the device selector, and select
-      **Edit configuration**.
+    * Debugging is based on a default launch configuration.
+      To customize this, click the drop-down button to the right
+      of the device selector, and select **Edit configuration**.
 
 ## Fast edit and refresh development cycle
 
 Flutter offers a best-in-class developer cycle enabling you to see the effect
-of your changes almost instantly with the _Stateful Hot Reload_ feature. See
-[Using hot reload](hot-reload) for details.
-
-## Advanced debugging
-
-### Debugging visual layout issues
-
-To debug a visual issue, run the app in **Debug** mode,
-and then open the Flutter inspector tool window using
-**View > Tool Windows > Flutter Inspector**.
-
-![Flutter Inspector Window]({% asset tools/android-studio/visual-debugging.png @path %})
-
-This offers many debugging tools; for details on these see
-[Debugging Flutter apps][].
-
-* **Enable select widget mode**: Select a widget on the device to inspect
-  in the [Flutter Inspector](/docs/development/tools/inspector).
-* **Refresh widget info**: Reload the current widget info.
-* **Show/hide performance overlay**: Toggle display of performance graphs
-  for the GPU & CPU threads.
-* **Toggle platform mode**: Toggle between rendering for Android or iOS.
-* **Show debug paint**: Add visual debugging hints to the rendering
-  displaying borders, padding, alignment, and spacers.
-* **Show paint baselines**: Cause each RenderBox to paint a line at each
-  of its text baselines.
-* **Enable slow animations**: Slow down animations to enable visual
-  inspection.
-
-Also available in the "more actions" menu:
-
-* **Show Repaint Rainbow**: Shows rotating colors on layers when repainting.
-* **Hide Debug Mode Banner**: Hides the debug banner even when running a
-  debug build.
-* **Highlight nodes displayed in both trees**: In the inspector, highlights
-  nodes shown in both the details and summary trees.
+of your changes almost instantly with the _Stateful Hot Reload_ feature.
+See [Hot reload](hot-reload) for details.
 
 ### Show performance data
 
-To view the performance data, including the widget rebuild information,
-start the app in **Debug** mode, and then open the Performance tool window
-using **View > Tool Windows > Flutter Performance**.
+{{site.alert.note}}
+  The Performance pane is not yet available in DevTools,
```

Comment:
This doesn't seem right. We should direct users to the devtools timeline page if they want to understand performance. The IntelliJ performance pane just directs users to that page to see more information.

Reply:
ok!